app/config/sculpin_site.yml

```diff
@@ -4,6 +4,11 @@ github_enabled: true
 default_date_format: "F jS, Y"
 
 authors:
+  dk:
+      name: Dominic Klein
+      title: Developer
+      bio: I work at shopware and do on-site as well as online trainings for developers. Besides that I look after our enterprise customers with focus on technical subjects.
```

Comment:
Shopware the Company is uppercase, shopware the product is lowercase.

Sorry, I',m forever spoiled by @tiagojsag.